### ISSUE TITLE
Centos-7.2 installation - missing ppc64le packages + install script wrong error message

### ISSUE THREAD

**Issue by grooverdan** [moby/moby#23882]
Error during installation:

```
#  curl -fsSL https://get.docker.com/ | sh
Error: you are not using a 64bit platform.
Docker currently only supports 64bit platforms.
```

Environment:

```
[root@dyn423 ~]# uname -a
Linux dyn423 4.7.0-rc3-next-20160617 #4 SMP Thu Jun 23 10:47:19 AEST 2016 ppc64le ppc64le ppc64le GNU/Linux
[root@dyn423 ~]# more /etc/redhat-release
CentOS Linux release 7.2.1511 (AltArch)
```

Centos with custom kernel in a KVM VM

So it is a 64bit VM so the bug is the script isn't generating the right error message.

Feature request - can Centos packages for ppc64le be added to https://yum.dockerproject.org/repo/main/centos/7/Packages/

Thank you.

**Comment by tophj-ibm** [comment]
Okay, I see the issue with the install script. 

It's worth mentioning that RHEL/Centos doesn't officially ship/support Docker on power currently, so I'm not sure if this is a good feature to get into atm.

**Comment by justincormack** [comment]
We do not provide binaries for ppc64le via `get.docker.com` yet. You can install from source, and some packages can be built.

**Comment by AkihiroSuda** [comment]
ppc64le binaries are no longer released since 18.09: https://download.docker.com/linux/static/stable/ppc64le/
